src/com/cdd/bao/template/ModelSchema.java

```diff
@@ -84,7 +84,8 @@ datastructure in a local file (using Turtle format), or to push & pull between a
 	public static final String IS_EXCLUDE = "isExclude"; // indicates a value refers to something to exclude
 	public static final String IS_WHOLEBRANCH = "isWholeBranch"; // indicates a value refers to an entire branch, not just one term
 	public static final String IS_EXCLUDEBRANCH = "isExcludeBranch"; // indicates a value refers to an entire branch to exclude
-	
+	public static final String IS_CONTAINER = "isContainer"; // a value thus specified means that it is not explicitly selected
```

Comment:
These four terms are linked to `Specify`. Should it be it's responsibility to define them?

Reply:
You're right.  They should be in the enum IMHO.